LaunchUI.sh

```diff
@@ -8,8 +8,8 @@
 # The tag "gui" is searched for as a base image for creating this container.
 
 docker run -d -it \
```

Comment:
No issue with this line I just don't know where to put this comment. I found that when I try running the container to view the UI there is currently an issue with a missing `progress` module. Not sure where the modules are being imported/installed at for this specific container.

```
(pipeline-uJrnX6UJ) erikhanson@pop-os:~/Documents/Dev/TeamBeeCapstoneProject$ docker logs pipeline 
Traceback (most recent call last):
  File "app.py", line 5, in <module>
    import progress
ModuleNotFoundError: No module named 'progress'
```

Reply:
Huh, this is running the LaunchUI.sh script? I'm not sure what that error refers to because that isn't imported in app.py. The dependencies (which is only Flask) are in the Dockerfile at docker/GUI/Dockerfile

Reply:
I was able to get the UI to run through the docker container. I had to modify the contents of the `Dockerfile` in the `docker` directory. Below is what the new Dockerfile looks like.

```
FROM python:3.7.7
RUN python -m pip install \
        Flask \
        progress

WORKDIR /pipeline/ui
CMD ["python", "app.py"]
```

Reply:
Just pushed those changes.